content/webapp/components/HTMLSerializers/HTMLSerializers.tsx

```diff
@@ -8,7 +8,7 @@ import DownloadLink from '@weco/common/views/components/DownloadLink/DownloadLin
 import { getMimeTypeFromExtension } from '@weco/common/utils/mime';
 import styled from 'styled-components';
 
-const DocumentType = styled.span.attrs({ className: 'no-margin' })`
```

Comment:
Was it just not needed?

Reply:
Ah yeah, I checked it locally – span already has zero margin, so this was having no effect.